python/grass/temporal/temporal_algebra.py

```diff
@@ -1234,10 +1231,7 @@ def check_stds(self, input, clear=False, stds_type=None, check_type=True):
         """
         if isinstance(input, str):
             # Check for mapset in given stds input.
-            if input.find("@") >= 0:
-                id_input = input
-            else:
-                id_input = input + "@" + self.mapset
+            id_input = input if input.find("@") >= 0 else input + "@" + self.mapset
```

Comment:
```suggestion
            id_input = input if "@" in input else f"{input}@{self.mapset}"
```

Reply:
All of these I'll do later, as it will conflict with work already done and waiting to be filed. (There's more to change)